docs/manual/troubleshooting.tex

```diff
@@ -838,6 +852,16 @@ \subsection{Build the Type Annotations compiler\label{building-compiler}}
 %END LATEX
 
 \item
+{\bf [Mac OSX 10.5 or later]} It is recommended to set the \code{\$JAVA\_HOME} variable to \</usr/libexec/java\_home>, just export \code{\$JAVA\_HOME} in file \code{.bash\_profile} or \code{.profile}.
```

Comment:
Is this guaranteed to work even if the user has installed his or her own JDK, such as JDK 9?
It's easy to write a hard-code command that works in the usual case, but it is better to write one that always works.
If this only works in certain situations, then at least say that.

Also, rather than "it is recommended", say why.